src/cubic_loader/qlik/ods_tables.py

```diff
@@ -0,0 +1,15 @@
+CUBIC_ODS_TABLES = [
+    "EDW.DATE_DIMENSION",
+    "EDW.FARE_PROD_USERS_LIST_DIMENSION",
+    "EDW.FARE_PRODUCT_DIMENSION",
+    "EDW.MEDIA_TYPE_DIMENSION",
+    "EDW.OPERATOR_DIMENSION",
+    "EDW.RIDE_TYPE_DIMENSION",
+    "EDW.ROUTE_DIMENSION",
+    "EDW.STOP_POINT_DIMENSION",
+    "EDW.TRANSIT_ACCOUNT_DIMENSION",
+    "EDW.TXN_STATUS_DIMENSION",
+    "EDW.CARD_DIMENSION",
+    "EDW.DEVICE_DIMENSION",
+    "EDW.USE_TRANSACTION",
+]
```

Comment:
This list controls which ODS-Qlik tables are loaded from the error and archive S3 buckets.